src/resources/filters/quarto-post/dashboard.lua

```diff
@@ -738,41 +738,11 @@ function render_dashboard()
         end
       end
     }, {
-      -- todo: dark mode
       Meta = function(meta)
```

Comment:
This seemed weird to me - I'm using a Lua filter just to copy a filter param (with fully-resolved logo) to Pandoc metadata. Is this a normal thing to do, or should I have changed the document metadata that goes into Pandoc? I couldn't figure out how to do so.

Reply:
> more reasonably I think a diagram could be drawn of all the metadatas and how they feed into each other, like our big process diagram

Yes, we direly need it. Earlier today, I made a separate comment about the precise location where our Markdown envelope gets created (and how it impacts shortcode resolution).

We need a similar thing about how metadata is assembled and handled.